mhctools/cli/script.py

```diff
@@ -57,6 +62,6 @@ def main(args_list=None):
     }
     epitope_collection = predictor.predict(input_dictionary)
     df = epitope_collection.to_dataframe()
-    print(df)
```

Comment:
Printing this DataFrame is sometimes the only output I care about (e.g. when checking if there are any strong MHC binding subsequences within one particular sequence), so I'd rather leave this as a print (or at least promote the log statement to `info`).